build_artifacts/v2/v2.0/v2.0.0/v2.0.0-beta/dirs/etc/maxdome/git_clone.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+METADATA=/opt/ml/metadata/resource-metadata.json
+
+# Extract the required fields from meta data stored in opt/ml/metadata.
+dataZoneProjectRepositoryName=$(grep -o '"DataZoneProjectRepositoryName":"[^"]*"' "$METADATA" | cut -d ':' -f 2 | tr -d '"')
+
+#User directory on which the user will have full access to . 
+MD_USER="sagemaker-user"
+USER_DIRECTORY="/home/${MD_USER}"
+
+# Configuring git credential helper to use AWS creds when connecting to Codecommit Repo. 
+git config --global credential.helper '!aws codecommit credential-helper $@' 
+git config --global credential.UseHttpPath true
+
+if  [ ! -d $USER_DIRECTORY/$dataZoneProjectRepositoryName ]; then
+    git clone  https://git-codecommit.$AWS_REGION.amazonaws.com/v1/repos/$dataZoneProjectRepositoryName $USER_DIRECTORY/$dataZoneProjectRepositoryName 
```

Comment:
We probably wanna do nohup to avoid latency.

Reply:
The idea is that we run the Script itself in nohup: basically something like `nohup docker Exec  git_clone.sh`